src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/NewlineAtEndOfFileCheck.xml

```diff
@@ -49,7 +49,7 @@
  &lt;/p&gt;</description>
          <properties>
             <property default-value="" name="fileExtensions" type="java.lang.String[]">
-               <description>Specify the file type extension of the files to check.</description>
+               <description>Specify the file type extension of files to process.</description>
```

Comment:
```suggestion
               <description>Specify the file extensions of the files to process.</description>
```

Reply:
`file type extension` is nonsense.

Reply:
done as separarte commit, as update it big.

additionally to show that removal of javadoc is well useful to avoid doing all of this updates in javadoc manually